scaleway/provider_test.go

```diff
@@ -282,12 +283,14 @@ func TestAccScalewayProvider_SSHKeys(t *testing.T) {
 			metaProd, err := buildMeta(ctx, &metaConfig{
 				terraformVersion: "terraform-tests",
 				httpClient:       tt.Meta.httpClient,
+				forceProjectID:   "11111111-1111-1111-1111-111111111111",
```

Comment:
Forcing project_id in these tests break them. Try to record provider tests `TestAccScalewayProvider.*`